registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidator.java

```diff
@@ -323,8 +338,24 @@ private boolean isValidOperator(RegOsiDto regOsi, String registrationId,
 							StatusMessage.PASSWORD_OTP_FAILURE);
 				}
 			} else {
+				String process=registrationStatusDto.getRegistrationType();
+			
+				if(!subProcesses.contains(process)) {
+				InfoResponseDto infoResponseDto=packetManagerService.getInfo(registrationId);
```

Comment:
u should not call containerinfo here. See the whole idea of getInfo is to decide priority which will be decided by PriorityBasedPacketManagerService. If stage level priority is set PriorityBasedPacketManagerService will make sure you get the data based on priority. So u should call getMetaInfo() to get the meta info and write logic on same.
This code might not be required when u call get metainfo directly.